consensus/src/dag/dag_network.rs

```diff
@@ -74,15 +74,16 @@ impl Responders {
     }
 }
 
+pub type RpcWithFallbackResult = (Author, anyhow::Result<DAGRpcResult>);
```

Comment:
nit: I'd probably call it ResultWithResponder and put author after (or use a struct)